app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -202,6 +202,14 @@ def build_metric_id(type, resource, metric_id)
       "#{type}I~R~[#{resource[:middleware_server][:feed]}/#{resource[:nativeid]}]~#{type}T~#{metric_id}"
     end
 
+    def self.update_alert(*args)
```

Comment:
It looks like #9511 which builds on top of this one indeed does:
https://github.com/ManageIQ/manageiq/pull/9511/files#diff-280d41e58c3a9c503d6bd1f110c873b1R171